packages/app/src/server/models/user.js

```diff
@@ -483,6 +483,16 @@ module.exports = function(crowi) {
     return usernameUsable;
   };
 
+  userSchema.statics.isRegisterableEmail = async function(email) {
+    let isEmailUsable = false;
+
+    const userData = await this.findOne({ email });
+    if (userData != null) {
+      isEmailUsable = true;
+    }
+    return isEmailUsable;
+  };
+
```

Comment:
sorry, plese code like the following
```suggestion
  userSchema.statics.isRegisterableEmail = async function(email) {
    let isEmailUsable = true;

    const userData = await this.findOne({ email });
    if (userData) {
      isEmailUsable = false;
    }
    return isEmailUsable;
  };

```